packages/app/src/common/validate-password.ts

```diff
@@ -0,0 +1,38 @@
+import zxcvbn, { ZXCVBNResult, ZXCVBNScore } from 'zxcvbn';
```

Comment:
Added a check here, on thomas' suggestion, to check for bip39 seeds, so no one enters that as a password
https://github.com/blockstack/stacks-wallet/blob/beta/app/crypto/validate-password.ts